documents/coredocument.go

```diff
@@ -586,6 +587,26 @@ func (cd *CoreDocument) Signatures() (signatures []coredocumentpb.Signature) {
 	return signatures
 }
 
+// AddUpdateLog adds a log to the model to persist an update related meta data such as author
```

Comment:
This description seems outdated since we removed the log message?

Reply:
but its still a log.. IMO